# 所有锁状态消息（针对一个板子）
uint8 board_address                # 板地址 (1-32)
bool[] lock_states                 # 所有锁的状态数组 (最多24个锁)
